build.gradle

```diff
@@ -248,110 +145,12 @@ allprojects {
-            header = createPythonLicenseWith(rootProject.file('LICENSE_SHORT'))
-            source = pythonTarget
-            strictCheck = true
-        }
-        licensePythonCheck.configure {
-            dependsOn tasks.matching { it.name == 'generate' }
-        }
-        def licensePythonFormat = tasks.register('licensePythonFormat', LicenseFormat) {
-            header = createPythonLicenseWith(rootProject.file('LICENSE_SHORT'))
-            source = pythonTarget
-            strictCheck = true
-        }
-        licensePythonFormat.configure {
-            dependsOn tasks.matching { it.name == 'generate' }
-        }
-
-        def registerPythonFormatTask = {
-            String taskName, String moduleName, String settingsFileFlagName, String... extraArgs ->
-            def task = tasks.register(taskName, Exec) {
-                workingDir projectDir
-                commandLine rootProject.file('.venv/bin/python')
-                args "-m", moduleName, settingsFileFlagName, rootProject.file('pyproject.toml').absolutePath
-                args extraArgs
-                pythonTarget.getFiles().forEach {
-                    args projectDir.relativePath(it)
-                }
-            }
-            task.configure {
-                dependsOn rootProject.tasks.named('pipInstall')
-                dependsOn tasks.matching { it.name == 'generate' }
-            }
-            return task
-        }
-
-        def isortPythonCheck = registerPythonFormatTask('isortPythonCheck', 'isort', '--settings-file', '--check-only')
-        isortPythonCheck.configure {
-            dependsOn licensePythonCheck
-        }
-        def isortPythonFormat = registerPythonFormatTask('isortPythonFormat', 'isort', '--settings-file')
-        isortPythonFormat.configure {
-            dependsOn licensePythonFormat
-        }
-
-        def blackPythonCheck = registerPythonFormatTask('blackPythonCheck', 'black', '--config', '--check')
-        blackPythonCheck.configure {
-            dependsOn licensePythonCheck
-            dependsOn isortPythonCheck
-        }
-        def blackPythonFormat = registerPythonFormatTask('blackPythonFormat', 'black', '--config')
-        blackPythonFormat.configure {
-            dependsOn licensePythonFormat
-            dependsOn isortPythonFormat
-        }
-    }
-
-    tasks.register('format') {
-        dependsOn tasks.matching { it.name == 'generate' }
-        dependsOn tasks.matching { it.name =~ /spotless.*Apply/ }
-        dependsOn tasks.matching { it.name =~ /.*PythonFormat/ }
-    }
-    tasks.named('check').configure {
-        dependsOn tasks.matching { it.name == 'licensePythonCheck' }
-        dependsOn tasks.matching { it.name == 'isortPythonCheck' }
-        dependsOn tasks.matching { it.name == 'blackPythonCheck' }
     }
 }
+// TODO: remove this once the CI routinely enforces formatting checks.
+tasks.matching { it.name =~ /spotless.*Check/ }.configureEach {
+    enabled = false
+}
```

Comment:
@postamar unfortunately bringing this back didn't fix it, because `gradle check` is running `spotlessSql`, apparently: https://github.com/airbytehq/airbyte/actions/runs/6750162945/job/18352320885#step:7:149

Reply:
But maybe it is just the regex? it is running `:airbyte-cdk:spotlessGroovyGradleCheck` for example

Reply:
This is hardly obvious but `spotlessSql` is a predecessor task to `spotlessSqlCheck`. The former generates the re-formatted files somewhere in the `build` directory and the latter compares them to the original files. Similarly, `spotlessSqlApply` also depend on `spotlessSql` but will overwrite the original files with the reformatted files instead of comparing them.